realsense2_camera/src/ros_sensor.cpp

```diff
@@ -94,7 +94,14 @@ void RosSensor::setParameters(bool is_rosbag_file)
     // So, during init of the node, forcefully disabling the HDR upfront and update it with new values.
     if((!is_rosbag_file) && supports(RS2_OPTION_HDR_ENABLED))
     {
-        set_option(RS2_OPTION_HDR_ENABLED, false);
+        try
+        {
+            set_option(RS2_OPTION_HDR_ENABLED, false);
```

Comment:
Why should we fail here?

Reply:
This is a call to fw, can eventually fail.
Should not, be better to protect it, as we do for every call to FW.

Reply:
On this case I suggest a macro / function that do this is 1 place.
```
if supported
   try
      ...
   catch
      ...
```
as you use it in several places

Reply:
macro added